src/components/Slider/Slider.module.scss

```diff
@@ -0,0 +1,78 @@
+.RootContainer {
```

Comment:
Nice, the styles are look good. A preference I do have is nesting the class names so its easier at first glance to see whats being used and where. I'll add an example below. Don't necessarily have to follow this but if you think its worthwhile feel free to update :)

So based on lines 27 - 47 in `RangeSilder.tsx`, we have a number of elements, all wrapped in div tags with RootContainer as the class name. SliderRoot, SliderTrack etc are nested within. I like to write the style file in similar fashion.

```
.RootContainer {
  //// styles
  .LightRootContainer {
    //// styles
  }

  .SliderRoot {
    //// styles
    .SmallRoot {
      //// styles
    }

    .LightRoot {
      //// styles
    }

    .SliderTrack {
      //// styles
      .SliderRange {
        //// styles
      }
    }
  }
}
```

Something like that. A suggestion but no worries :)

Reply:
Done Shifted styles to match their structure in the component

Reply:
Sweet its all good. This style helps me quite a lot, was sharing just in case it made sense to you too, not needed at all though only a personal touch

Reply:
I'd like to get them nested before approving this one - nested styles are much more maintainable. Let me know if you need assistance here @Tweemo.

Reply:
Managed to get them nested and looking good, some handy learning was done :D